fury/ui/containers.py

```diff
@@ -92,9 +120,10 @@ def resize(self, size):
         ----------
         size : (float, float)
             Panel size (width, height) in pixels.
-
         """
         self.background.resize(size)
+        button_coords = (int(size[0] - self.resize_button.size[0]), 0)
+        self.update_element(self.resize_button, button_coords)
 
```

Comment:
Do we implement resizing of contained components along with this PR or do we do it in a separate PR? @skoudoro @antrikshmisri . Unfortunately, I am not able to recall.

Reply:
I can't recall either but I think it would make sense to do that in a seperate PR.